config/checkstyle/checkstyle.xml

```diff
@@ -391,12 +391,6 @@
     </module>
 
     <module name="InvalidJavadocPosition"/>
-
-    <module name="MissingJavadocMethodCheck">
-      <property name="minLineCount" value="1"/>
-      <property name="allowMissingPropertyJavadoc" value="true"/>
-      <property name="ignoreMethodNamesRegex" value="(set.*|get.*)"/>
-    </module>
```

Comment:
I found this to be silly, I don't believe in the "every method needs to be documented" philosophy